share/sv.po

```diff
@@ -122,10 +116,10 @@ msgstr ""
 
 msgid ""
 "Boolean flag for activity indicator. Defaults to on if STDOUT is a tty, off "
-"if it is not. Disable with --noprogress."
+"if it is not. Disable with --no-progress."
 msgstr ""
 "Flagga för om en aktivitets-indikator skall användas. Normalt på om STDOUT "
-"är en tty, och av annars. Inaktivera med --noprogress."
+"är en tty, och av annars. Inaktivera med --no-progress."
```

Comment:
`tty` -> `TTY`?

Reply:
I agree. Also for English that whould probably be better. Fixed.